ios/FluentUI/Vnext/Drawer/SlideOverPanel.swift

```diff
@@ -144,6 +154,10 @@ struct MSFSlideOverPanel<Content: View>: View {
         }
         return minValue
     }
+
+    private func shadowColor(_ isPrimary: Bool = true) -> Color {
+        return isPrimary ? tokens.shadow1Color : tokens.shadow2Color
```

Comment:
I think this is not needed. Can you just use them directly on the .shadow modifier?